twitter.js

```diff
@@ -34,6 +35,17 @@ const baseHeaders = {
   Accept: "application/json"
 };
 
+function formatOauthRes(txt){
```

Comment:
We don't need this custom function we're already using the querystring library which has a querystring.parse method we can use instead.

Reply:
And the same comments apply for the getAccessToken method.